SConstruct

```diff
@@ -103,6 +103,17 @@ AddOption(
     help='Version of avtranscoder library to link (needed only if compile on Windows).'
 )
 
+# Get gtest install path
+AddOption(
+    '--gtest',
+    dest='gtest',
+    type='string',
+    nargs=1,
+    action='store',
+    metavar='DIR',
+    help='Path to gtest library (used for the tests).'
```

Comment:
It's the path of the root deployment of gtest, not really the library (the .a/.so/.dll file).